sopel/modules/instagram.py

```diff
@@ -0,0 +1,95 @@
+# coding=utf-8
+"""
+instagram.py - API-key-less Instagram module for Sopel
+Copyright 2018, Author Name withheld
```

Comment:
Hahaha :joy:

If you really don't want to put a handle here (`kwaaak` would be fine, or your IRC nick, or whatever pseudonym), apply this suggestion and re-squash.

```suggestion
Copyright 2018, Sopel contributors
```

Reply:
Always something :grin: